src/main/java/cat/udl/eps/softarch/demo/domain/Business.java

```diff
@@ -0,0 +1,69 @@
+package cat.udl.eps.softarch.demo.domain;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Entity
+@Table(name = "business")
+@Data
+@EqualsAndHashCode
+public class Business {
```

Comment:
This is a class Business will use to get authenticated. Do like it is done for Customer, extend the User class and use a String as the identifier